packages/yuudachi/src/interactions/moderation/status.ts

```diff
@@ -0,0 +1,27 @@
+import { ApplicationCommandOptionType } from 'discord-api-types/v10';
+
+export const StatusCommand = {
+	name: 'status',
```

Comment:
Best case this should really be a sub command under /report, top-level `status` can be confusing, even with the description

Reply:
The real problem with this is, the mods can't hide subcommands iirc, I can maybe make the `/reports` to have an subcommand for `lookup` and one for `status`

Reply:
Yeah thatd be a good solution

Reply:
Changed in 81f8374